freetextresponse/tests.py

```diff
@@ -260,6 +261,23 @@ def test_compute_score(self, credit):
             {'value': credit.value, 'max_value': Credit.full.value},
         )
 
+    @ddt.data(Credit.zero, Credit.half, Credit.full)
+    def test_compute_score_error(self, credit):
+        # pylint: disable=protected-access
+        """
+        Tests _compute_score's error handling
```

Comment:
@jlikhuva Yeah, I think that's what @caseylitton is pointing out; the comment implies that this handles more than it actually does. We should only be catching that specific exception in this case; let's update the docs to reflect what we're catching.